docs/index.rst

```diff
@@ -1,408 +1,66 @@
-=====================
-urllib3 Documentation
-=====================
+urllib3
+=======
 
 .. toctree::
    :hidden:
+   :maxdepth: 2
 
-   pools
-   managers
-   security
-   helpers
-   exceptions
-   collections
-   contrib
-   recipes
+   user-guide
+   advanced-usage
+   reference/index
+   contributing
 
+Urllib3 is a *sanity-friendly* HTTP client for Python. It helps you avoid the
+inevitable heartache and frustration of lower-level HTTP clients. It is the
+magic beneath the fantastic `Requests <http://python-requests.org/>`_ library.
```

Comment:
:D I'll go in and start changing small things like this inline later, don't have time to dive in deep yet.

Also if you could make a "Who uses urllib3" section, that's be great! (requests, pip, etc to start, and there are a bunch of companies who use it straight-up that I'll get a list for later)